_posts/2021-01-12-github-application-running-on-faasd-behind-inlets.md

```diff
@@ -0,0 +1,508 @@
+---
+layout: post
+title: How to expose your Github application running locally on faasd to the internet ?
+description: In this guide, we are going to demonstrate how we can build our Github Application and run it locally on our faasd instance then expose it to the internet using inlets to be able to Github can send events to it
```

Comment:
"how we can" -> "how to"
Github application > "GitHub App"

Reply:
done